tag_features.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+"""
+This script takes a file containing a collection of one or more feature 
+definitions, that is pointed to by the -f flag and a tag name pointed to by 
+the -t flag.  The tag is added to each feature (if that feature does not 
+already have that tag.  Results are written back to the same feature file.
+"""
+
+import os.path
+import json
+import argparse
+from collections import defaultdict
+from utils.feature_write_utils import write_all_features
+
+
+parser = argparse.ArgumentParser(description=__doc__, 
+                                 formatter_class=argparse.RawTextHelpFormatter)
+parser.add_argument("-f", "--feature_file", dest="feature_file", 
+                    help="Features collection file to be tagged", metavar="FILE",
+                    required=True)
+parser.add_argument("-t", "--tag", dest="tag", help="Tag to add to all features", 
+                    metavar="TAG", required=True)
+parser.add_argument("-r", "--remove", dest="remove", action='store_true',
+                    help="Use this tag to signal removing a tag instead of adding")
```

Comment:
Can you change this to "Use this flag to signal.." ? Just to keep argument flags separate from the feature tags you're trying to modify here?